packages/indexer-common/src/indexer-management/client.ts

```diff
@@ -222,6 +276,15 @@ const SCHEMA_SDL = gql`
 
     storeDisputes(disputes: [POIDisputeInput!]!): [POIDispute!]
     deleteDisputes(allocationIDs: [String!]!): Int!
+
+    createAllocation(deploymentID: String!, amount: String!): CreateAllocationResult!
+    closeAllocation(id: String!, poi: String, force: Boolean): CloseAllocationResult!
```

Comment:
For `closeAllocation` and `reallocateAllocation`, I like that a POI can be specified, but I think that the default behaviour should be that `indexer-agent` attempts to fetch a valid POI from `graph-node`, and if a POI is provided, it's effectively an override.

Reply:
Oh, I see your comment in the PR:

```**: The PoI parameter is optional; if provided it will be compared against a PoI resolved from the graph-node and the request
will be rejected if they do not match. The user can also specify --force (-f) to bypass the PoI accuracy check and force it to send the close transaction with the supplied Poi.
```

Ignore my comment :)